### DISCUSSION TITLE
Client npm package

### DISCUSSION THREAD

**Discussion by anirudh-singh01** [sahilatahar/Code-Sync#11] [General]
While installing the npm package in the client folder using the npm install command it gets stuck at half of the process and not installing properly. Does anyone know how to solve it? I have tried performing the same thing multiple times but the same thing is happening.
![Screenshot 2024-06-13 182538](https://github.com/sahilatahar/Code-Sync/assets/129600428/c2417eec-d211-49a1-9a5d-1f2f04676a23)

**Comment by sahilatahar** [comment]
@anirudh-singh01 Delete node_modules and lock file then try to install.

**Reply by anirudh-singh01** [reply]
It was a node version problem, which was solved by upgrading it to the latest version of Node js
